src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/AggregateCompletionListProvider.cs

```diff
@@ -56,7 +58,7 @@ public AggregateCompletionListProvider(IEnumerable<CompletionListProvider> compl
 
                     cancellationToken.ThrowIfCancellationRequested();
                 }
-                catch (Exception ex) when (ex is not TaskCanceledException)
+                catch (Exception ex) when (ex is not OperationCanceledException)
```

Comment:
This was an oversight on my end from the previous PR

Reply:
I thought about this when I was reviewing the other PR, but didn't mention it. I should have. I apologize.

Reply:
I failed fast 😄